"webservices"/metro/wsit/... 4.0.0 doesn't work in GlassFish

webservices-osgi.jar, webservices-osgi-api.jar, etc in version 4.0.0 have been created using the Apache Maven Bundle Plugin 5.1.6. 

This version of the plug-in adds tons of "strange" Java SE packages to the import section, which subsequently fails on GF.

To reproduce: in GF 7 master, in nucleus/parent/pom.xml

set in `webservices.version>4.0.0-M4</webservices.version>` the version to 4.0.0. Build GF, and subsequently start it. See the horror unfold in the logs.

```
INFO: Create bundle provisioner class = class com.sun.enterprise.glassfish.bootstrap.osgi.BundleProvisioner.
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishMain.main(GlassFishMain.java:73)
	at com.sun.enterprise.glassfish.bootstrap.ASMain.main(ASMain.java:30)
Caused by: org.jvnet.hk2.config.ConfigPopulatorException: Failed to parse domain.xml
	at org.glassfish.config.support.DomainXml.throwParseError(DomainXml.java:166)
	at org.glassfish.config.support.DomainXml.run(DomainXml.java:114)
	at org.jvnet.hk2.config.ConfigurationPopulator.populateConfig(ConfigurationPopulator.java:34)
	at org.glassfish.hk2.bootstrap.HK2Populator.populateConfig(HK2Populator.java:83)
	at com.sun.enterprise.module.common_impl.AbstractModulesRegistryImpl.populateConfig(AbstractModulesRegistryImpl.java:190)
	at com.sun.enterprise.module.bootstrap.Main.createServiceLocator(Main.java:249)
	at org.jvnet.hk2.osgiadapter.HK2Main.createServiceLocator(HK2Main.java:96)
	at com.sun.enterprise.glassfish.bootstrap.osgi.EmbeddedOSGiGlassFishRuntime.newGlassFish(EmbeddedOSGiGlassFishRuntime.java:88)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishRuntimeDecorator.newGlassFish(GlassFishRuntimeDecorator.java:43)
	at com.sun.enterprise.glassfish.bootstrap.osgi.OSGiGlassFishRuntime.newGlassFish(OSGiGlassFishRuntime.java:66)
	at com.sun.enterprise.glassfish.bootstrap.GlassFishMain$Launcher.launch(GlassFishMain.java:89)
	... 6 more
Caused by: org.glassfish.config.support.DomainXml$NoBackupException: No usable configuration file at /Users/arjan/eclipse/workspace/glassfish/appserver/tests/tck/concurrency/target/glassfish7/glassfish/domains/domain1/config
	at org.glassfish.config.support.DomainXml.getAlternativeDomainXml(DomainXml.java:228)
	at org.glassfish.config.support.DomainXml.run(DomainXml.java:110)
	... 15 more
	Suppressed: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.webservices.connector [215]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.webservices.config.RegistryLocationInjector
	name=registry-location
	contracts={org.glassfish.webservices.config.RegistryLocationInjector,org.jvnet.hk2.config.ConfigInjector}
	scope=jakarta.inject.Singleton
	qualifiers={org.jvnet.hk2.config.InjectionTarget}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=@connector-resource-jndi-name={optional,datatype\:java.lang.String,leaf},keyed-as={org.glassfish.webservices.config.RegistryLocation},key={@connector-resource-jndi-name},target={org.glassfish.webservices.config.RegistryLocation},Bundle-SymbolicName={org.glassfish.main.webservices.connector},Bundle-Version={7.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.webservices.connector [215]], State = [NEW],1218519094)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=320
	locatorId=0
	identityHashCode=200404000
	reified=false)

		at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2262)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:452)
		at org.jvnet.hk2.config.DomDocument$InjectionTargetFilter.matches(DomDocument.java:160)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.getDescriptors(ServiceLocatorImpl.java:342)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.getDescriptors(ServiceLocatorImpl.java:384)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.getBestDescriptor(ServiceLocatorImpl.java:392)
		at org.jvnet.hk2.config.DomDocument.buildModel(DomDocument.java:111)
		at org.jvnet.hk2.config.ConfigModel.parseValue(ConfigModel.java:937)
		at org.jvnet.hk2.config.ConfigModel.<init>(ConfigModel.java:852)
		at org.jvnet.hk2.config.DomDocument.buildModel(DomDocument.java:90)
		at org.jvnet.hk2.config.DomDocument.getModelByElementName(DomDocument.java:138)
		at org.jvnet.hk2.config.ConfigParser.handleElement(ConfigParser.java:141)
		at org.jvnet.hk2.config.ConfigParser.parse(ConfigParser.java:77)
		at org.jvnet.hk2.config.ConfigParser.parse(ConfigParser.java:71)
		at org.glassfish.config.support.DomainXml.parseDomainXml(DomainXml.java:284)
		at org.glassfish.config.support.DomainXml.run(DomainXml.java:99)
		... 15 more
	Caused by: com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.webservices.connector [215]], State = [NEW]
		at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
		at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
		at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2254)
		... 31 more
	Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.webservices.connector [215](R 215.0): missing requirement [org.glassfish.main.webservices.connector [215](R 215.0)] osgi.wiring.package; (&(osgi.wiring.package=jakarta.xml.ws)(version>=4.0.0)(!(version>=5.0.0))) [caused by: Unable to resolve org.glassfish.metro.webservices-api-osgi [247](R 247.0): missing requirement [org.glassfish.metro.webservices-api-osgi [247](R 247.0)] osgi.wiring.package; (osgi.wiring.package=java.io)] Unresolved requirements: [[org.glassfish.main.webservices.connector [215](R 215.0)] osgi.wiring.package; (&(osgi.wiring.package=jakarta.xml.ws)(version>=4.0.0)(!(version>=5.0.0)))]
		at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4398)
		at org.apache.felix.framework.Felix.startBundle(Felix.java:2308)
		at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:1006)
		at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
		at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:185)
		... 33 more
```

--- a/wsit/pom.xml
+++ b/wsit/pom.xml
@@ -138,7 +138,8 @@
                 <plugin>
                     <groupId>org.apache.felix</groupId>
                     <artifactId>maven-bundle-plugin</artifactId>
-                    <version>5.1.6</version>
+                    <!-- 5.1.6 drops (default) compatibility with OSGi R6 -->
+                    <version>5.1.4</version>
                     <configuration>
                         <instructions>
                             <_noextraheaders>true</_noextraheaders>

